webpack.config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
         ]
       },
       {
-        test: /\.s(a|c)ss$/,
+        test: /.(css|scss)$/,
```

Comment:
please add a new test for CSS instead of changing this one? @nkpremices

Reply:
Noted @truestbyheart